pkg/packagekit/updates.jsx

```diff
@@ -634,6 +968,26 @@ class OsUpdates extends React.Component {
                 .fail(this.handleLoadError);
     }
 
+    callTracer(state) {
+        this.setState({ tracerRunning: true });
+        python.spawn(callTracerScript, null, { err: "message", environ: ['LC_ALL=C.UTF-8'], superuser: "require" })
+                .then(output => {
+                    const nextState = { tracerRunning: false, restartableServices: JSON.parse(output) };
+                    if (state)
+                        nextState.state = state;
+
+                    this.setState(nextState);
+                })
+                .fail((exception, data) => {
+                    console.error(`Tracer failed: "${JSON.stringify(exception)}", data: "${JSON.stringify(data)}"`);
+                    // When tracer fails, act like it's not available (demand reboot after every update)
```

Comment:
:+1: That's exactly the EAFP approach I mentioned above, then you can drop the "is it available" check beforehand.